great_expectations/datasource/batch_kwargs_generator/databricks_batch_kwargs_generator.py

```diff
@@ -3,6 +3,9 @@
 from great_expectations.datasource.batch_kwargs_generator.batch_kwargs_generator import (
     BatchKwargsGenerator,
 )
+from great_expectations.datasource.util import (
+    get_or_create_spark_session as get_or_create_spark_session_v012,
```

Comment:
Why is this being renamed? If it is api-named, it should be referring to "batch kwargs api" or "v2"

Reply:
Fixed (in next commit) -- it will be `get_or_create_spark_session_v2` (to signify the API naming for version 2, Legacy).